flair/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 # global variable: cache_root
 cache_root = Path(os.getenv("FLAIR_CACHE_ROOT", Path(Path.home(), ".flair")))
+cache_root.mkdir(exist_ok=True, parents=True)
```

Comment:
why is this necessary?

Reply:
I ran into problems with the cache folder not being created correctly, but thinking of it again, I think that was due to different errors and this line is not necessary. I'll remove it again

Reply:
thanks for double-checking!